kotlin-sdk-core/src/commonMain/kotlin/io/modelcontextprotocol/kotlin/sdk/types.kt

```diff
@@ -1051,6 +1098,34 @@ public enum class Role {
     assistant,
 }
 
+/**
+ * Optional annotations for the client.
+ * The client can use annotations to inform how objects are used or displayed.
+ */
+@Serializable
+public data class Annotations(
+    /**
+     * Describes who the intended customer of this object or data is.
+     */
+    val audience: List<Role>?,
```

Comment:
Why is there a default `null` in one case, but not here?

Reply:
I only added the `null`s in the other classes to be source-compatible with the latest version. I personally prefer when there are default `null`s when an attribute is nullable, but they are currently missing on some classes, for example `Resource` and `ResourceTemplate`.

 I will add them here too.

Also, can I add the missing `null` defaults on `Resource` and `ResourceTemplate`?

Reply:
Sorry, for the delay btw. I am on vacation so I'm not very attentive...